scripts/style.sh

```diff
@@ -69,10 +73,18 @@ fi
 
 # Format C-ish sources only
 \%\.(h|m|mm|cc)$% p
-' | xargs clang-format -style=file $options \
-  | grep "<replacement " > /dev/null
+'
+)
+needs_formatting=false
+for f in $files; do
```

Comment:
This construct will eventually fail if the number files gets too large; given how many files we have now I'm surprised this even works.

If it does work for now with all files then I guess this is okay, but we should probably revisit at some point.